### PR TITLE
Add example for validator class with custom data

### PR DESCRIPTION
**Pull request by D-Marc1**
Show that you need to change the constructor and create the proper type.

### CODE REVIEW

File: content/guides/validator/introduction.md [hunk 1]
```diff
@@ -232,26 +232,43 @@ You can also manually construct the class instance and pass any arguments you li
...
 
-Following is an example of using the validator classes outside of the HTTP request.
+Your validator class would look like the following:
 
 ```ts
-import { validate } from '@ioc:Adonis/Core/Validator'
-import CreateUser from 'App/Validators/CreateUserValidator'
+import { schema } from '@ioc:Adonis/Core/Validator'
+
+interface Country {
```
Comment (thetutlage) [inline content/guides/validator/introduction.md:248]: Don't understand the point of adding these interfaces

File: content/guides/validator/introduction.md [hunk 2]
```diff
@@ -232,26 +232,43 @@ You can also manually construct the class instance and pass any arguments you li
...
+    new CreateUserValidator({
       countries: fetchAllowedCountries(),
       states: fetchAllowedStates()
     })
   )
 })
 ```
 
-Following is an example of using the validator classes outside of the HTTP request.
+Your validator class would look like the following:
```
Comment (thetutlage) [inline content/guides/validator/introduction.md]: The sentence must include `validator classes usage outside of the HTTP request`

**Review by thetutlage — CHANGES_REQUESTED** [review]

File: content/guides/validator/introduction.md (hunk 1 above)
Comment (D-Marc1) [reply content/guides/validator/introduction.md:248]: I added them for type hinting, as this seemed like what the example's type was looking for. I can remove the type hinting if you want though and just use `any`, if that's your preference, to make it more concise.

File: content/guides/validator/introduction.md (hunk 2 above)
Comment (D-Marc1) [reply content/guides/validator/introduction.md]: Changed the wording. Let me know if you need any changes to my modified wording.

File: content/guides/validator/introduction.md (hunk 1 above)
Comment (thetutlage) [reply content/guides/validator/introduction.md:248]: Yeah, I think the main purpose is to not show the type of data you can pass. It is to show that you can pass data via constructor. So having types just make the whole example look crowded

**Review by thetutlage — CHANGES_REQUESTED** [review]

**Pull request closed without merging**